books/free-programming-books-subjects.md

```diff
@@ -916,6 +917,13 @@ Kerridge (PDF) (email address *requested*, not required)
 * [Version Control with Subversion](https://svnbook.red-bean.com/index.en.html) - Ben Collins-Sussman, Brian W. Fitzpatrick, C. Michael Pilato
 
 
+### Virtual Reality
+
+* [AUGMENTED REALITY](https://library.oapen.org/bitstream/id/f922c42d-ad74-4af1-953c-02c7518f37a7/628401.pdf) - José María Ariso (Ed.)
+* [VIRTUAL REALITY](http://msl.cs.uiuc.edu/vr/vrbook.pdf) - Steven M. LaValle
```

Comment:
should use this url: http://lavalle.pl/vr/
We don't use ALL CAPS titles.
I'm wondering why use "virtual reality" for "augmented reality" and "exgended reality". Might these be better under "graphics programming" or "game development? The is a virtual worlds book in Game development.
Be sure to check the free-programming-books-lint output for linter errors. Alphabetical order and spacing are the most common problems.